Traceback (most recent call last):
  File "/camcan/anaconda3/envs/chiara_env/lib/python3.8/site-packages/seaborn/palettes.py", line 222, in color_palette
    palette = _ColorPalette(palette)
  File "/camcan/anaconda3/envs/chiara_env/lib/python3.8/site-packages/matplotlib/colors.py", line 346, in to_rgb
    return to_rgba(c)[:3]
  File "/camcan/anaconda3/envs/chiara_env/lib/python3.8/site-packages/matplotlib/colors.py", line 189, in to_rgba
    rgba = _to_rgba_no_colorcycle(c, alpha)
  File "/camcan/anaconda3/envs/chiara_env/lib/python3.8/site-packages/matplotlib/colors.py", line 260, in _to_rgba_no_colorcycle
    raise ValueError(f"Invalid RGBA argument: {orig_c!r}")
ValueError: Invalid RGBA argument: 'fpn#F7EA23'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/camcan/FPNflexiblehubs/FPNflexiblehubs/flexiblehubsrunPoolSubjsCC.py", line 45, in <module>
    studyforrest.stage4_analysis(chunklen, corr=1)
  File "/camcan/FPNflexiblehubs/FPNflexiblehubs/PoolSubjs.py", line 420, in stage4_analysis
    custom_p = sns.set_palette(sns.color_palette(colors))
  File "/camcan/anaconda3/envs/chiara_env/lib/python3.8/site-packages/seaborn/palettes.py", line 224, in color_palette
    raise ValueError(f"Could not generate a palette for {palette}")
ValueError: Could not generate a palette for <map object at 0x7f5e3f2ba6a0>
